src/utils/domUtils.ts

```diff
@@ -20,13 +20,13 @@ export module Utils {
       return bbox;
     }
 
-    export var POLYFILL_TIMEOUT_MILLISECONDS = 1000 / 60; // 60 fps
+    export var REQUEST_ANIMATION_FRAME_TIMEOUT_MILLISECONDS = 1000 / 60; // 60 fps
 
     export function requestAnimationFramePolyfill(fn: () => void) {
```

Comment:
What is a `Polyfill` do you think this is an appropriate name now that the constant is named differently?